graphdatascience/session/aura_api.py

```diff
@@ -114,7 +115,9 @@ def create_session(
 
         self._check_resp(response)
 
-        return SessionDetails.from_json(response.json()["data"])
+        raw_json: Dict[str, Any] = response.json()
+
+        return SessionDetails.from_json(raw_json["data"], raw_json.get("errors", []))
```

Comment:
What if `data` key is missing? Do we handle it somewhere else?

Reply:
nope. 
Arguably, we could also expect "errors" to be present. 
If its not, thats a breaking aura-api change.

Especially as this change is already live in Aura we could always expect errors to be present